Failing test: Rules Management - Bundled Prebuilt Rules Integration Tests - Serverless Env - Complete License.x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/bundled_prebuilt_rules_package/trial_license_complete_tier/install_latest_bundled_prebuilt_rules·ts - Rules Management - Prebuilt Rules - Bundled Prebuilt Rules Package @ess @serverless @skipInQA install_bundled_prebuilt_rules should install prebuilt rules from the package that comes bundled with Kibana

A test failed on a tracked branch

```
JestAssertionError: expect(received).toBeGreaterThan(expected)

Expected: > 0
Received:   0
    at Context.<anonymous> (install_latest_bundled_prebuilt_rules.ts:77:82)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Object.apply (wrap_function.js:73:16) {
  matcherResult: {
    message: '\x1B[2mexpect(\x1B[22m\x1B[31mreceived\x1B[39m\x1B[2m).\x1B[22mtoBeGreaterThan\x1B[2m(\x1B[22m\x1B[32mexpected\x1B[39m\x1B[2m)\x1B[22m\n' +
      '\n' +
      'Expected: > \x1B[32m0\x1B[39m\n' +
      'Received:   \x1B[31m0\x1B[39m',
    pass: false
  }
}
```

First failure: [CI Build - main](https://buildkite.com/elastic/kibana-on-merge/builds/43306#018eaa8d-2d29-4696-b2a0-55496745e558)

<!-- kibanaCiData = {"failed-test":{"test.class":"Rules Management - Bundled Prebuilt Rules Integration Tests - Serverless Env - Complete License.x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/bundled_prebuilt_rules_package/trial_license_complete_tier/install_latest_bundled_prebuilt_rules·ts","test.name":"Rules Management - Prebuilt Rules - Bundled Prebuilt Rules Package @ess @serverless @skipInQA install_bundled_prebuilt_rules should install prebuilt rules from the package that comes bundled with Kibana","test.failCount":2}} -->

--- a/x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/common/prebuilt_rules_package/air_gapped/install_bundled_package.ts
+++ b/x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/prebuilt_rules/common/prebuilt_rules_package/air_gapped/install_bundled_package.ts
@@ -15,6 +15,7 @@ import { ALL_SAVED_OBJECT_INDICES } from '@kbn/core-saved-objects-server';
 import { FtrProviderContext } from '../../../../../../../ftr_provider_context';
 import {
   deleteAllPrebuiltRuleAssets,
+  deletePrebuiltRulesFleetPackage,
   getPrebuiltRulesStatus,
   installPrebuiltRulesPackageByVersion,
 } from '../../../../../utils';
@@ -31,27 +32,21 @@ export default ({ getService }: FtrProviderContext): void => {
   /* attempt to install it from the local file system. The API response from EPM provides
   /* us with the information of whether the package was installed from the registry or
   /* from a package that was bundled with Kibana */
-  //
-  // FLAKY: https://github.com/elastic/kibana/issues/180087
-  describe.skip('@ess @serverless @skipInServerlessMKI Install bundled package', () => {
+  describe('@ess @serverless @skipInServerlessMKI Install bundled package', () => {
     beforeEach(async () => {
       await deleteAllRules(supertest, log);
       await deleteAllPrebuiltRuleAssets(es, log);
+      await deletePrebuiltRulesFleetPackage({ es, supertest, retryService: retry, log });
     });
 
     it('should list `security_detection_engine` as a bundled fleet package in the `fleet_package.json` file', async () => {
       const configFilePath = path.resolve(REPO_ROOT, 'fleet_packages.json');
       const fleetPackages = await fs.readFile(configFilePath, 'utf8');
-
       const parsedFleetPackages: PackageSpecManifest[] = JSON5.parse(fleetPackages);
 
-      const securityDetectionEnginePackage = parsedFleetPackages.find(
-        (fleetPackage) => fleetPackage.name === 'security_detection_engine'
+      expect(parsedFleetPackages).toEqual(
+        expect.arrayContaining([expect.objectContaining({ name: 'security_detection_engine' })])
       );
-
-      expect(securityDetectionEnginePackage).not.toBeUndefined();
-
-      expect(securityDetectionEnginePackage?.name).toBe('security_detection_engine');
     });
 
     it('should install prebuilt rules from the package that comes bundled with Kibana', async () => {
@@ -77,9 +72,12 @@ export default ({ getService }: FtrProviderContext): void => {
 
       // Verify that status is updated after package installation
       const statusAfterPackageInstallation = await getPrebuiltRulesStatus(es, supertest);
-      expect(statusAfterPackageInstallation.stats.num_prebuilt_rules_installed).toBe(0);
+
       expect(statusAfterPackageInstallation.stats.num_prebuilt_rules_to_install).toBeGreaterThan(0);
-      expect(statusAfterPackageInstallation.stats.num_prebuilt_rules_to_upgrade).toBe(0);
+      expect(statusAfterPackageInstallation.stats).toMatchObject({
+        num_prebuilt_rules_installed: 0,
+        num_prebuilt_rules_to_upgrade: 0,
+      });
     });
   });
 };

